impl/sql/sequenced/sequenced.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sequenced stores a list of objects that have been sequenced.
+package sequenced
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/gob"
+	"errors"
+	"fmt"
+
+	"github.com/google/keytransparency/core/sequenced"
+	"github.com/google/keytransparency/core/transaction"
+)
+
+const (
+	insertMapRowExpr = `INSERT INTO Maps (MapID) VALUES (?);`
+	countMapRowExpr  = `SELECT COUNT(*) AS count FROM Maps WHERE MapID = ?;`
+	insertExpr       = `
+	INSERT INTO Sequenced (MapID, Epoch, Data)
+	VALUES (?, ?, ?);`
+	readExpr = `
+	SELECT Data FROM Sequenced
+	WHERE MapID = ? AND Epoch = ?;`
+	latestExpr = `
+	SELECT Epoch, Data FROM Sequenced
+	WHERE MapID = ? 
+	ORDER BY Epoch DESC LIMIT 1;`
+)
+
+var (
+	createStmt = []string{
+		`
+	CREATE TABLE IF NOT EXISTS Maps (
+		MapID   BIGINT NOT NULL,
+		PRIMARY KEY(MapID)
+	);`,
+		`
+	CREATE TABLE IF NOT EXISTS Sequenced (
+		MapID   BIGINT      NOT NULL,
+		Epoch   BIGINT      NOT NULL,
```

Comment:
Please make this `INTEGER` to be consistent with the other database table definitions.